### PR TITLE
HWKMETRICS-115 Create a Gatling load testing scenario

### PR DESCRIPTION
**Pull request by tsegismont**
Create a Gatling load testing scenario

### CODE REVIEW

**Comment by slandelle** [comment]
:beers:

**Comment by tsegismont** [comment]
Thanks for your help @slandelle :wink:  I'll open the beer tonight, I promise! :beer:

**Comment by tsegismont** [comment]
@slandelle by the way, I haven't checked if the intermediary stats can be sent to Hawkular Metrics. Maybe with our Influx compatibility layer it would work OTB. Eventually we could have a native Hawkular Metrics.

File: load-tests/README.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+:type: article
+
+A simple load testing tool for Hawkular Metrics
+
+== Why a Maven project?
+
+Because it makes it easier for developers to fire a load test:
+there's no need to download, extract and configure PATH.
+
+== The scenario
```
Comment (jsanda) [inline load-tests/README.adoc]: The README should indicate that you need to start up Cassandra and Hawkular separately.

Comment (tsegismont) [inline load-tests/README.adoc]: Done. See commit 2fbdf97

**Comment by jsanda** [comment]
I am ok with merging this. It is a good initial effort. @stefannegrea, @mwringe wdyt?

**Comment by mwringe** [comment]
I am ok with merging as well. Is there any plans to include more than just the gauges as part of this?

**Comment by jsanda** [comment]
> I am ok with merging as well. Is there any plans to include more than just the gauges as part of this?
I think this is just an initial PoC that demonstrates how to get up and running with Gatling. We definitely will test more than just gauges. 

I should also point out that I am not necessarily advocating for/against Gatling. Getting it into master increases the likelihood of others exploring it and providing more feedback.

**Comment by tsegismont** [comment]
Le 02/06/2015 18:08, mwringe a écrit :
> I am ok with merging as well. Is there any plans to include more than
> just the gauges as part of this?

In the future we will make it possible to include availability in the 
mix of values.

**Comment by tsegismont** [comment]
Le 02/06/2015 18:21, jsanda a écrit :
> I should also point out that I am not necessarily advocating for/against
> Gatling. Getting it into master increases the likelihood of others
> exploring it and providing more feedback.

To me, the ability to simulate a great number of clients from the same 
machine is a very good advantage.

Other than that, what's really missing when starting a load test is 
"metrics from Metrics" :)

I would like to include Codahale's library soon and start exposing 
performance indicators. Like metrics ingested/min.

**Pull request merged**